examples/tutorial_derive/04_01_enum.rs

```diff
@@ -10,7 +10,11 @@ struct Cli {
 
 #[derive(Copy, Clone, PartialEq, Eq, PartialOrd, Ord, ValueEnum)]
 enum Mode {
+    /// Run swiftly
     Fast,
+    /// Crawl slowly but steadily
+    ///
+    /// This paragraph is ignored because there is no long help text for possible values.
     Slow,
 }
 
```

Comment:
While your intention is to demonstrate how doc comments behave, please reflect these changes in the builder tutorial as we keep these in sync.